conn.go

```diff
@@ -1055,6 +1055,33 @@ func (c *Conn) Create(path string, data []byte, flags int32, acl []ACL) (string,
 	return res.Path, err
 }
 
+func (c *Conn) CreateContainer(path string, data []byte, flags int32, acl []ACL) (string, error) {
+	if err := validatePath(path, flags&FlagSequence == FlagSequence); err != nil {
+		return "", err
+	}
+	if flags != FlagContainer {
+		return "", errors.New("flags not support container node")
+	}
+
+	res := &createResponse{}
+	_, err := c.request(opCreateContainer, &CreateContainerRequest{path, data, acl, flags}, res, nil)
+	return res.Path, err
+}
+
+// ttl: ms
+func (c *Conn) CreateTTL(path string, data []byte, flags int32, acl []ACL, ttl int64) (string, error) {
+	if err := validatePath(path, flags&FlagSequence == FlagSequence); err != nil {
+		return "", err
+	}
+	if flags != FlagWithTTL && flags != FlagSequenceWithTTL {
```

Comment:
shouldn't this be tested as a bites, e.g. `flags&FlagWithTTL != FlagWithTTL`?

Reply:
At first I think bit 0 is for Ephemeral，bit 1 is for Sequence，and bit 2 is for TTL，but the value 4(bit 2) is FlagContainer, so I was confused, and listed all the flag values. For example, if I use bit 2 for FlagTTL, when I create container node, I have to use FlagTTL .

Reply:
In other way, container node will be deleted in some condition, so it can alse be thinked as ttl. Perhaps bit 2 for FlagTTL is well. I'll modify it.

Reply:
> @einxie could you follow up on the above comments please

You're right. Bit 2 for FlagTTL is well. I have modifed it.